string Class   
uint16 centerX
uint16 centerY 
float64 centerX_3
float64 centerY_3
float64 centerZ_3
float64 angleX
float64 angleY
